components/tools/python.xml

```diff
@@ -89,9 +89,9 @@ build_year = "${python_build_year}"
         <delete dir="../target/downloads/py/omero-py-${versions.omero-py}"/>
 
 
-        <download pkg="scripts" file="${versions.scripts}.tar.gz" expected="${versions.scripts-md5}"
+        <download pkg="scripts" file="$v{versions.scripts}.tar.gz" expected="${versions.scripts-md5}"
```

Comment:
```suggestion
        <download pkg="scripts" file="v${versions.scripts}.tar.gz" expected="${versions.scripts-md5}"
```

Reply:
Cheers. I had tested via a copied ant block to run _just_ `scripts` and failed to apply properly.